### ISSUE TITLE
Nested Empty Error 

### ISSUE THREAD

**Issue by PratikshaPanpaliya** [amancevice/flatsplode#8]
Not Working for 
{
	'id': '78e5b18c',
	'attrs': [],
	'keywords': [
		{'fizz':['fizza','fizzb']},
		{'buzz': []}
	]

}

output coming
![image](https://user-images.githubusercontent.com/56633686/165070208-986c754d-dac6-42b6-8963-dcd0426e69e1.png)
but this not expected output , shouldn't empty arrayy for attrs key showing correct but for nested empty array not able to handle

**Comment by amancevice** [comment]
Can you show me what you think the expected output should be in JSON?

**Comment by PratikshaPanpaliya** [comment]
Instead of empty array in keywords.buzz it should come None
![image](https://user-images.githubusercontent.com/56633686/165085158-465617d8-7a71-4059-ba47-4e5603f685cb.png)

**Comment by amancevice** [comment]
it would be helpful if you could manually create the actual JSON you think your example should explode into instead of a screen shot so I can write a test

**Comment by PratikshaPanpaliya** [comment]
This is json:
{
 'id': '78e5b18c',
'attrs': [],
'keywords': [
{'fizz':['fizza','fizzb']},
{'buzz': []}
]
}

**Comment by PratikshaPanpaliya** [comment]
Expected output


id              attrs   keywords.fizz  keywords.buzz
78e5b18c  None fizza                Nan
78e5b18c  None  fizzb              Nan
78e5b18c  None  Nan               None

**Comment by amancevice** [comment]
Right, I'm asking what do you think the result of calling `flatsplode()` on that object should be. In other words:

```python
from flatsplode import flatsplode

item = {
    'id': '78e5b18c',
    'attrs': [],
    'keywords': [
        {'fizz': ['fizza', 'fizzb']},
        {'buzz': []}
    ]
}
list(flatsplode(item)) = ???
```

**Comment by amancevice** [comment]
I think it should be:

```python
[
    {'id': '78e5b18c', 'attrs': None, 'keywords.fizz': 'fizza'},
    {'id': '78e5b18c', 'attrs': None, 'keywords.fizz': 'fizzb'},
    {'id': '78e5b18c', 'attrs': None, 'keywords.buzz': None},
]
```

Do you agree?

**Comment by PratikshaPanpaliya** [comment]
Yes. Sorry I was converting to data frame

**Comment by amancevice** [comment]
this should be fixed as of v0.3.1
